src/core/config/config.service.ts

```diff
@@ -91,6 +91,13 @@ export class ConfigService {
     };
   }
 
+  @Lazy() get passwordSecret() {
+    return Buffer.from(
+      this.env.string('PASSWORD_SECRET').optional('passwordSecret'),
```

Comment:
Let's just omit it by default as to not mess up existing databases
```suggestion
      this.env.string('PASSWORD_SECRET').optional(),
```